### ISSUE TITLE
requery button in UI empties form fields

### ISSUE THREAD

**Issue by pramsey** [CrunchyData/pg_featureserv#30]
This is true for both ordinary collections and functions, but hitting the requery button in the map view causes the form fields to become empty. Also clears the "bbox" check box. Makes it hard to iterate and fine tune values in the fields to experiment if you have non-trivial, multi-argument input.

**Comment by dr-jts** [comment]
Yes, agreed.   It's just this way to reduce amount of client and server-side code required.  But should be fairly easy to fix.
